specification/storage/resource-manager/Microsoft.Storage/stable/2025-01-01/storageTaskAssignments.json

```diff
@@ -652,6 +653,7 @@
           "enum": [
             "ValidateSubscriptionQuotaBegin",
             "ValidateSubscriptionQuotaEnd",
+            "Accepted",
```

Comment:
As agreed with Storage action team before, Storage Action team should make "provisioningState" property as `"modelAsString": true`  on a new API version (2025-01-01) to avoid breaking in the future.

Reply:
Added these changes, can you please help with approvals from SDK teams?

Reply:
I have already ping the JS/GO SDK owner in Teams channel for Storage Action.